analyzer/runtime/extract.go

```diff
@@ -390,6 +390,7 @@ func ExtractRound(blockHeader nodeapi.RuntimeBlockHeader, txrs []nodeapi.Runtime
 						if to, err = uncategorized.StringifySdkAddress(body.To); err != nil {
 							return fmt.Errorf("to: %w", err)
 						}
+						blockTransactionData.RelatedAccountAddresses[to] = true
```

Comment:
Correct, the caller is already registered as a related account [at the beginning](https://github.com/oasisprotocol/nexus/blob/9e33faed3fc86d90edd16892b51826a38cc569fc/analyzer/runtime/extract.go#L350-L350) of this (huge) func. So we're fine in terms of tracking related accounts.

It's a separate question whether we want to make the implicit value more explicit. E.g. one could easily argue that when faced with a tx with no `to`, Explorer would be more helpful if it showed the implied `to` value (= the sender), rather than nothing.
Tough call. So far, our guiding principle has been that the indexer just indexes; it does not mangle data irreversibly. According to that philosophy, we should keep storing and representing such txs as having no `to`, and then the frontend can render the implied value (possibly with a visual indicator that it's implied).